apps/calendar/js/views/event_base.js

```diff
@@ -281,6 +281,11 @@ EventBase.prototype = {
     // always remove loading initially (to prevent worst case)
     this.element.classList.remove(this.LOADING);
 
+    // Re-run the header font fit when it comes into view.
+    // Since the header is already in the markup on load and the view is hidden
+    // the font fit calculations will be wrong initially.
+    this.header.runFontFitSoon();
```

Comment:
ok, I needed to run "make" so that I see the change in Nightly. This definitely fixes it so please ignore my previous comment.